src/app/GitUI/UserControls/BlameControl.cs

```diff
@@ -644,16 +636,12 @@ private void blamePreviousRevisionToolStripMenuItem_Click(object sender, EventAr
                 return;
             }
 
-            // Try get actual parent revision, get popup if it does not exist.
-            // (The menu should be disabled if previous is not in grid).
-            GitRevision selectedRevision = _revisionGridInfo!.GetActualRevision(blameInfo.selectedRevision);
-
             // Origin line of commit selected is final line of the previous blame commit
             int finalLineNumberOfPreviousBlame = _lastBlameLine!.OriginLineNumber;
-            int originalLineNumberOfPreviousBlame = _gitBlameParser.GetOriginalLineInPreviousCommit(selectedRevision, blameInfo.filename, finalLineNumberOfPreviousBlame);
+            int originalLineNumberOfPreviousBlame = _gitBlameParser.GetOriginalLineInPreviousCommit(blameInfo.selectedRevision, blameInfo.filename, finalLineNumberOfPreviousBlame);
```

Comment:
0
This is the previous revision in grid.
I suggest this is separate or that the txt is changed.
Will not vote against though.